src/data/markdown/docs/02 javascript api/11 k6-ws/20-Params.md

```diff
@@ -6,10 +6,12 @@ excerpt: 'Used for setting various WebSocket request-specific parameters such as
 
 _Params_ is an object used by the WebSocket methods that generate WebSocket requests. _Params_ contains request-specific options like headers that should be inserted into the request.
 
-| Name                  | Type   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
-| --------------------- | ------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-| `Params.headers`      | object | Object with key-value pairs representing custom headers the user would like to add to the request.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
-| `Params.tags`         | object | Key-value pairs where the keys are names of tags and the values are tag values. Response time metrics generated as a result of the request will have these tags added to them, allowing the user to filter out those results specifically, when looking at results data.                                                                                                                                                                                                                                                                                                                          |
+| Name                  | Type   | Description |
+| --------------------- | ------ | ----------- |
+| `Params.compression`  | string | Empty by default, but when specified, WebSocket compression will be enabled with the specified algorithm. The only currently supported algorithm is `deflate`. |
```

Comment:
Empty by default, thus disabled. WebSocket compression will only be enabled when an algorithm is specified. The only currently supported algorithm is `deflate`.

Reply:
:thinking: WDYT about:
> WebSocket compression algorithm. It's empty by default, thus disabled, and will only be enabled when this option is specified. The only currently supported algorithm is `deflate`.

Reply:
Better.

Reply:
Resolved by https://github.com/grafana/k6-docs/pull/495/commits/268caeb8f30c7e52cd6b30649973a5d657bf8516

Reply:
I do think that the options documentation should:
1. first and foremost explain what it is for
2. what are the possible values
3. what is the default

in that order.
So for me @na-- suggestion is better, but something like 
> compression algorithm to be used by the WebSocket connection. The only supported algorithm is `deflate` and if the option is left unset it defaults to no compression.  

is even more concise IMO.

Reply:
Agree, resolved by https://github.com/grafana/k6-docs/pull/495/commits/a23595697396c19921c989b142830213c683a6d5, but please don't comment in resolved PR discussions (or unresolve them if you do), I almost missed this.